wxgui: Use correct (widget) with for word wrapping of warnings and errors

marisn opened the pull request:
Startup screen uses window width for word wrapping but actual space for text is smaller. This PR fixes incorrect word wrapping by using available width instead of window width.

PS. Any reason why here is not used StaticWrapText from gui_core?

petrasovaa commented:
I don't have any issues with word wrapping here, everything fits and wraps ok on Ubuntu, what platform do you use?. Also, GetClientSize is I think preferable than GetSize here, although I don't know if it matters in this specific case. Any reason why you use GetSize?

StaticWrapText might work too, I don't remember any specific reason why it's not there.

marisn commented:
> StaticWrapText might work too, I don't remember any specific reason why it's not there.

I replaced the widget with StaticWrapText and rebased on top of current master.
Here's an example how it looks like without this patch:
![GRASS_startup_wrap](https://user-images.githubusercontent.com/17123023/78636392-ed5b9200-78b0-11ea-8920-d94f1081303e.png)
